### ISSUE TITLE
As browser scollbar - HOW?

### ISSUE THREAD

**Issue by diomed** [noraesae/perfect-scrollbar#527]
Hi!

Can you please show me how could I use this as browser scrollbar, on my page:
Just like [this template](http://template.impala.itembridge.com/index1.html) does.

**Comment by TehSomeLuigi** [comment]
Just leaving this comment here, I'm not familiar with perfect-scrollbar yet, or even that well acquainted with CSS (I understand CSS but we never seem to get on too well).

But I've noticed that they seem to have wrapped their content in a div with the class name `impala-scroll-overlay` and then seem to have initialised perfect scrollbar on that div, seemingly like any other.

If this isn't true/helpful, I apologise, but hopefully it can shed some light on the situation.

I found this JavaScript in the template:
```
	if (navigator.userAgent.match(/(iPhone|iPod|iPad|Android|BlackBerry|Windows Phone)/)){
	  $('.impala-scroll-overlay').perfectScrollbar("destroy");
	  $('.impala-scroll-overlay').addClass('scroll-block');
	  $('.bg-video').find('video').remove();
	} 
	else {
		$('.impala-scroll-overlay').perfectScrollbar();
		$('.impala-scroll-overlay').removeClass('scroll-block');
	}
```
Again, this is not guaranteed, but I would say that it's essentially doing the following:

**If** the device is a mobile phone or tablet:
- Destroy (stop using) the Perfect Scrollbar
- Add the scroll-block CSS class to the wrapped content
which, with the following CSS,
```
.impala-wrap-home .scroll-block {
    height: 100%;
    overflow-y: auto;
}
```
, makes it so the content is normally scrollable (using the browser's default scrollbar)
- Remove the video (I suppose they must have a video somewhere, doesn't seem relevant in any case)

**Otherwise**, if not using one of those mobile devices:
- Initialise perfect scrollbar on the container
- Remove the scroll-block class, to suppress the browser's default scrollbar (because the overflow will revert to hidden)

I hope that makes some sort of sense; some of the explanation isn't 100% strictly correct.
Maybe it'll save the developer of perfect-scrollbar some time responding to this issue, and can serve as my gratitude, both for writing perfect-scrollbar, and for having a non-jQuery version.

Cheers.
